client/client.go

```diff
@@ -69,77 +74,56 @@ type User struct {
 	// be public (start with a capital letter)
 }
 
-// InitUser will be called a single time to initialize a new user.
 func InitUser(username string, password string) (userdataptr *User, err error) {
 	var userdata User
-	userdataptr = &userdata
 
-	//TODO: This is a toy implementation.
+	// TODO: This is a toy implementation.
 	userdata.Username = username
-	//End of toy implementation
 
 	return &userdata, nil
 }
 
-// GetUser is documented at:
-// https://cs161.org/assets/projects/2/docs/client_api/getuser.html
 func GetUser(username string, password string) (userdataptr *User, err error) {
 	var userdata User
 	userdataptr = &userdata
-
 	return userdataptr, nil
 }
 
-// StoreFile is documented at:
-// https://cs161.org/assets/projects/2/docs/client_api/store_file.html
```

Comment:
I totally agree that keeping this link synced is a PITA. However, is it worth having a stock comment saying something like "this function is documented in the Project 2 spec. Please go read those docs before implementing this function."?

I guess it is clearly implied, but given the questions that students have asked in the past, I think it is better to be explicit in an attempt to avoid students questions wherever possible?